types/styled-components/ts3.6/index.d.ts

```diff
@@ -75,59 +74,34 @@ type StyledComponentPropsWithAs<
     T extends object,
     O extends object,
     A extends keyof any
-> = StyledComponentProps<C, T, O, A> & { as?: C, forwardedAs?: C };
+> = StyledComponentProps<C, T, O, A> & { as?: C; forwardedAs?: C };
```

Comment:
Is there a reason not to support the same `forwardedAs` variance in the ts3.6 types?

Reply:
Oh, that's TS@3.6 not Styled Components@3.6. 😞 

Yup, I can update that too.

Reply:
Gah, lands squarely in a weird TODO section. Let's see how true that TODO comment is...